### ISSUE TITLE
Printer registration missing for exception Response.Error.E

### ISSUE THREAD

**Issue by ddickstein** [ocaml/ocaml-lsp#1084]
[This exception](https://github.com/ocaml/ocaml-lsp/blob/master/jsonrpc/src/jsonrpc.ml#L186) is missing a printer.

**Comment by rgrinberg** [comment]
> though that may mean taking yojson as a dependency in the jsonrpc library

Yes, this is what I was trying to avoid. I do have the registration in `jsonrpc-fiber`. If you think it's worth it, I can bring it back though.

**Comment by ddickstein** [comment]
At least for my use case (lsp rpc for other languages), I already take a yojson dependency, so introducing a dependency in jsonrpc doesn't matter to me. But if you'd prefer to leave it as is I can copy the implementation from jsonrpc-fiber.
